src/pages/blog/starlingx-ptp-multi-instance-features.md

```diff
@@ -0,0 +1,126 @@
+NIC
+
+Documentation for multi-instance configuration can be found here:
+
+https://docs.starlingx.io/system_configuration/kubernetes/configuring-ptp-service-using-the-cli.html#ptp-instance-configuration
+
+
+# Local GNSS Time Sources
+
+For performant PTP operation, the Grand Master clock requires a highly accurate timing source which 
+it can use to distribute time across the network. StarlingX 7.0 provides the ts2phc application 
+which takes the timestamps from an incoming GNSS signal and synchronizes the configured NICs with 
+that timing information. This means that a StarlingX node with a compatible NIC and connected GNSS 
+antenna can serve as a reliable Grand Master clock without the need for a dedicated external 
+appliance. This functionality is desirable for edge nodes as it can reduce the complexity of 
+delivering the time source across disparate locations, instead allowing each edge site to have easy 
+access to a reliable clock source. A node with a GNSS time source can then be configured with ptp4l
+to distribute that timing information to other local nodes.
+
+
+The diagram below depicts an example PTP topology which uses a GNSS time source to provide accurate 
+timing to several local nodes:
+
+![alt text](/img/ptp-t-gm.png)
+
+1. Incoming GNSS signal from an antenna connected to a compatible NIC. This serves to provide 
+frequency information to the local PHC on the NIC
+2. Time of Day information is extracted from the GNSS signal using ts2phc and is replicated to the
+secondary NIC
+3. The system clock is synchronized to the primary NIC PHC using phc2sys
+4. A ptp4l instance is configured per-NIC, allowing timing information to be transmitted out each 
+port
+5. Bi-directional SMA ports connecting the primary and secondary NICs transmit the Pulse Per Second
+frequency information from the GNSS signal to the secondary NIC, ensuring that both NICs are locked
+to the same frequency
+6. Time of day information is transmitted out each port by ptp4l. Frequency information is
+transmitted via syncE
+
+# Basic Synchronous Ethernet Support
+Synchronous Ethernet (syncE) can be deployed alongside PTP to improve the timing accuracy of the 
+network. SyncE accomplishes this by distributing physical layer frequency synchronization across the
+network which is traceable to the primary reference clock (PRC). This means that each physical layer
+segment is going “tick tock tick tock” at the same frequency as distributed from the PRC source. 
+
+The effect deploying syncE alongside PTP serves to improve the time synchronization performance by 
+reducing jitter and wander, improving holdover performance, and enabling accurate time 
+synchronization across a longer chain of nodes.
+
+SyncE functionality requires the use of NICs with syncE capable hardware and drivers.
+
+Configuring syncE in StarlingX can be accomplished using the “clock” ptp-instance type on a 

```

Comment:
```suggestion
You can configure syncE in StarlingX by using the [“clock”](https://docs.starlingx.io/system_configuration/kubernetes/instance-specific-considerations-d9d9509c79dd.html#clock) ptp-instance type on a 
```suggestion
Configuring syncE in StarlingX can be accomplished using the “clock” ptp-instance type on a 

```
```